src/wmr-middleware.js

```diff
@@ -339,6 +374,19 @@ export const TRANSFORMS = {
 					spec = `/@npm/${meta.module}${meta.path ? '/' + meta.path : ''}`;
 				}
 
+				const modSpec = spec.replace('../', '/').replace('./', '/');
```

Comment:
I don't think this is a good solution, if anyone knows a better one feel free to let me know